app/main.py

```diff
@@ -1 +1,8 @@
-# write your code here
+def copy_file(command: str) -> None:
+    text = command.split(" ")
+    if text[0] != "cp":
+        return
+    if text[1] == text[2]:
+        return
```

Comment:
You able to merge this two conditions
```suggestion
    if text[0] != "cp" or text[1] == text[2]:
        return
```